cdap-app-fabric/src/main/java/co/cask/cdap/config/ConsoleSettingsStore.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright © 2015 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.config;
+
+import com.google.common.annotations.VisibleForTesting;
+import com.google.inject.Inject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+
+/**
+ * Console Settings Store Management.
+ */
+public class ConsoleSettingsStore {
```

Comment:
It seems like I saw similar class. May be create wrapper that takes config type as param? or make it abstract class?

Not sure it will be cleaner though - use your judgement :)